tests/test_trailedSourceFilter.py

```diff
@@ -0,0 +1,85 @@
+import unittest
+from lsst.ap.association import TrailedSourceFilterTask
+import numpy as np
+import pandas as pd
+import lsst.afw.image as afwImage
+import lsst.utils.tests
+
+
+class TestTrailedSourceFilterTask(unittest.TestCase):
+
+    def setUp(self):
+        """Create sets of diaSources and diaObjects.
+        """
+        rng = np.random.default_rng(1234)
+        scatter = 0.1 / 3600
+        self.nSources = 5
+        self.diaSources = pd.DataFrame(data=[
```

Comment:
Please include a note here about how many long trailed sources this test will generate: it's not at all obvious from the `5.5*idx` approach which of them will be long enough to be filtered.

Reply:
Updated the setUp doc string to explicitly list the trail lengths in arcseconds.